images/scripts/lib/bootc.Containerfile

```diff
@@ -2,8 +2,11 @@ ARG base_image
 FROM $base_image
 
 # pre-install the distro version, which is useful for testing extensions and manual experiments
+# also pre-install ws and test dependencies
 RUN \
+    dnf update -y --exclude='kernel*' && \
```

Comment:
Is that a known issue, sounds like it could use a `# HACK` unless we never want to support it?

Reply:
It's at least not necessary for us to support. I had to add it because the original container has a too old selinux-policy -- the rpm build in the mock of centos-9-stream depended on a newer version. I haven't reported the kernel failure yet, but added to my TODO list.